file-server/app.py

```diff
@@ -0,0 +1,149 @@
+        vo_url = vo.generate_voiceover(
+            str(edit_text), f"./static/{user_id}/audio/output_voiceover.mp3"
+        )
+        mg.create_video(
+            vo_url,
+            image_folder=f"./static/{user_id}/images",
+            output_file=f"./static/{user_id}/movie/output_video_with_audio.mp4",
+        )
+        return jsonify(
+            {"video_url": f"/static/{user_id}/movie/output_video_with_audio.mp4"}
+        )
+
+
+@app.route("/images", methods=["POST", "GET"])
+def images():
+    """
+    Takes user_id from json body and returns the urls of the images (from local storage).
+    If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
+        json_data = json.loads(request.data.decode("utf-8"))
+        user_id = json_data["user_id"]
+        if os.path.exists(f"file-server/static/{user_id}/images"):
+            imgs = os.listdir(f"./static/{user_id}/images")
+            return jsonify(
+                {
+                    "user_id": user_id,
+                    "image_urls": [
+                        f"/static/{user_id}/images/{image}" for image in imgs
+                    ],
+                }
+            )
+        else:
+            return jsonify({"error": "Images not found."})
+
+
+@app.route("/voiceover", methods=["POST"])
+def voiceover():
+    """
+    Takes user_id from json body and returns the url of the voiceover (from local storage).
+    If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
+        json_data = json.loads(request.data.decode("utf-8"))
+        user_id = json_data["user_id"]
+        if os.path.exists(f"file-server/static/{user_id}/audio/output_voiceover.mp3"):
+            return jsonify(
+                {
+                    "user_id": user_id,
+                    "voiceover_url": f"/static/{user_id}/audio/output_voiceover.mp3",
+                }
+            )
+        else:
+            return jsonify({"error": "Voiceover not found."})
+
+
+@app.route("/movie", methods=["POST"])
+def movie():
+    """
+    Takes user_id from json body and returns the url of the video (from local storage). If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
+        json_data = json.loads(request.data.decode("utf-8"))
+        user_id = json_data["user_id"]
+        if os.path.exists(
+            f"file-server/static/{user_id}/movie/output_video_with_audio.mp4"
+        ):
+            return jsonify(
+                {
+                    "user_id": user_id,
+                    "video_url": f"/static/{user_id}/movie/output_video_with_audio.mp4",
+                }
+            )
+        else:
+            return jsonify({"error": "Video not found."})
+
+
+
+@app.route("/reset", methods=["POST"])
+def reset():
+    """
+    Deletes all files in static/images, static/audio, static/movie.
+    """
+    data = json.loads(request.data.decode("utf-8"))
+    user_id = data["user_id"]
```

Comment:
use `dict.get("keyname", "defaultvalue")` method for a failsafe, just in case this isn't included in the request. You can also implement a helper function to do just this and check if that particular default was returned.